core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/appRoutesModels.scala

```diff
@@ -39,14 +39,16 @@ final case class GetAppResponse(kubernetesRuntimeConfig: KubernetesRuntimeConfig
                                 proxyUrls: Map[ServiceName, URL],
                                 diskName: Option[DiskName],
```

Comment:
I couldn't find any unit tests for these models (to add verification of AppType). Is there a test I should update to cover these changes?

Reply:
you could update [HttpRoutesSpec](https://github.com/DataBiosphere/leonardo/blob/develop/http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/http/api/HttpRoutesSpec.scala) to include a test case for appType in listApp and getApp endpoints

Reply:
Thanks @gpcarr. Looking at those tests, I think perhaps it already covers this change because of this sort of test. `getAppResponse` will include the new field.

````
it should "get app" in {
    Get("/api/google/v1/apps/googleProject1/app1") ~> routes.route ~> check {
      status shouldEqual StatusCodes.OK
      validateRawCookie(header("Set-Cookie"))
      responseAs[GetAppResponse] shouldBe getAppResponse
    }
  }

Reply:
this is covered...the test is using the same `ListAppResponse.fromCluster` helper, which is already updated